lighthouse-core/lib/page-functions.js

```diff
@@ -420,6 +420,21 @@ function wrapRequestIdleCallback(cpuSlowdownMultiplier) {
   };
 }
 
+const getNodeInfoString = `function getNodeInfo(elem) {
```

Comment:
🚲 🏠  and feel free to dismiss if you've already discussed this to arrive at the name, but if this is supposed to directly match the `node` details type in the future. WDYT about `getNodeDetails`?

Reply:
I like that 💯